docs/source/motivation.rst

```diff
@@ -29,45 +29,54 @@ I've used Celery_ professionally for years and my growing frustration
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Locks and rate limiting      | Yes      | No          | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Task prioritization          | Yes      | No [#prio]_ | No           | No [#prio]_  |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Delayed tasks                | Yes      | Yes [#]_    | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Cronlike scheduling          | No       | Yes         | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Chaining / Pipelining        | Yes      | Yes         | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Result storage               | Yes      | Yes         | Yes          | Yes          |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Code auto-reload             | Yes      | No          | No           | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| RabbitMQ support             | Yes      | Yes         | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Redis support                | Yes      | Yes         | Yes          | Yes          |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| In-memory broker support     | Yes      | No          | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Greenlet execution model     | Yes       | Yes         | Yes          | No           |
```

Comment:
Minor nitpick: There's an extra space in the Dramatiq column that makes the row formatting inconsistent with the other rows.

Reply:
Thanks @rpkilby !

Reply:
🙇